webform_civicrm.module

```diff
@@ -64,6 +64,12 @@ function webform_civicrm_menu() {
  * Implements hook_form_alter().
  */
 function webform_civicrm_form_alter(&$form, &$form_state, $form_id) {
+  if ($form_id == 'webform_edit_form' && !empty($form['webform_ui_elements'])
+    && !empty((int) preg_grep('/^civicrm_/', array_keys($form['webform_ui_elements'])))) {
+    \Drupal::service('civicrm')->initialize();
+    \CRM_Core_Resources::singleton()->addCoreResources();
+  }
+
```

Comment:
🤔 does this run on the Build tab? or is this for the component edit form?  Do we need the CiviCRM resources on that page? Or does it fix weird AJAX insertion of assets when the offcanvas form opens?

Reply:
Would it be "easier/better" to check if the webform has the CiviCRM handler versus preg_grep over each array key?

Reply:
yes, the above loads civicrm resources on the build tab so that contact element works fine when opened on the same page via AJAX. I've replaced the above condition to use handler check but it seems too large. Do you have any short suggestions?

Reply:
Looks good to me! I know loading assets over AJAX doesn't always work right, especially I would imagine it does not for CiviCRM resources.